agency_api/models.py

```diff
@@ -106,6 +107,8 @@ class HPJobApplication(models.Model):
     phone_number = models.BigIntegerField(null=False)
     email = models.EmailField(null=False, max_length=200)
     job = models.ForeignKey(JobPosting, on_delete=models.CASCADE)
+    is_pending = models.BooleanField(null=False, default=False)
```

Comment:
Shouldn't this default to True?

Reply:
Hiring a health care professional isn't working. The reason is the HealthCareProfessional model has a gender field with a max length of 1, but the HPJobApplicationModel allows 10 characters, so it gets an error when it tries to create a HealthCareProfessional. I think yesterday we talked about switching the way we store gender to use 'Male' and 'Female' instead of 'M' and 'F'. You would just need to allow for 10 characters in the HealthCareProfessional model